src/pages/task/Metadata.tsx

```diff
@@ -19,22 +18,23 @@ export const Metadata: React.FC<{
   const task = data ? data.task : null;
   const taskAnalytics = useTaskAnalytics();
 
-  const spawnHostLink = get(task, "spawnHostLink");
-  const createTime = get(task, "createTime");
-  const finishTime = get(task, "finishTime");
-  const hostId = get(task, "hostId");
-  const hostLink = get(task, "hostLink");
-  const startTime = get(task, "startTime");
-  const estimatedStart = get(task, "estimatedStart");
-  const timeTaken = get(task, "timeTaken");
-  const baseCommit = get(task, "revision", "").slice(0, 10);
-  const reliesOn = get(task, "reliesOn");
-  const baseTaskMetadata = get(task, "baseTaskMetadata");
-  const baseTaskDuration = get(baseTaskMetadata, "baseTaskDuration");
-  const baseTaskLink = get(baseTaskMetadata, "baseTaskLink");
+  const spawnHostLink = task?.spawnHostLink;
+  const createTime = task?.createTime;
+  const finishTime = task?.finishTime;
+  const hostId = task?.hostId;
+  const hostLink = task?.hostLink;
+  const startTime = task?.startTime;
+  const estimatedStart = task?.estimatedStart;
+  const timeTaken = task?.timeTaken;
+  const baseCommit = task?.revision.slice(0, 10);
+  const reliesOn = task?.reliesOn;
+  const baseTaskMetadata = task?.baseTaskMetadata;
+  const ami = task?.ami;
+  const baseTaskDuration = baseTaskMetadata?.baseTaskDuration;
+  const baseTaskLink = baseTaskMetadata?.baseTaskLink;
```

Comment:
some of these had default values which should be preserved. for example, `task?.revision.slice(0, 10);` will throw an error when revision is `undefined`. the default value should be `const revision = task?.revision ?? ""` then `revision.slice(0,10)`

Reply:
Thanks nice catch, Accidentally  took it out